versiondb/client/verify.go

```diff
@@ -109,6 +121,20 @@ func VerifyChangeSetCmd(defaultStores []string) *cobra.Command {
 
 			commitInfo := buildCommitInfo(storeInfos, lastestVersion)
 
+			if len(saveSnapshot) > 0 {
+				// write multitree metadata
+				metadata := memiavl.MultiTreeMetadata{
+					CommitInfo: &commitInfo,
+				}
+				bz, err := metadata.Marshal()
+				if err != nil {
+					return err
+				}
+				if err := writeFileSync(filepath.Join(saveSnapshot, memiavl.MetadataFileName), bz); err != nil {
```

Comment:
maybe just export [writeSnapshot](https://github.com/crypto-org-chain/cronos/blob/21d00240f5b58cd2af913d6b406ac15719e0f65f/memiavl/multitree.go#L369) to use in here

Reply:
I think it's alright